mysql-test/t/insert_select.opt

```diff
@@ -0,0 +1 @@
+--loose-innodb-vtq
```

Comment:
It enables innodb-vtq plugin for I_S (all plugins are disabled by default in mtr).